includes/minishell.h

```diff
@@ -2,6 +2,9 @@
 # define MINISHELL_H
 
 #include <unistd.h>
```

Comment:
norminette will annoye you with the identation, should be # include (a space is missing)